electrum/commands.py

```diff
@@ -1322,6 +1322,34 @@ async def reverse_swap(self, lightning_amount, onchain_amount, wallet: Abstract_
             'onchain_amount': format_satoshis(onchain_amount_sat),
         }
 
+    @command('n')
+    async def convert_currency(self, from_amount, from_ccy = None, to_ccy = 'BTC'):
+        """Converts the given amount of currency to another using the
+        configured exchange rate source.
+        """
```

Comment:
Note that fx is disabled by default, so we should probably give a friendly error message:
```python
if not self.daemon.fx.is_enabled():
    raise Exception("FX is disabled. To enable, run: 'electrum setconfig use_exchange_rate true'")
```